cocotb/drivers/avalon.py

```diff
@@ -43,13 +45,14 @@
 
 
 class AvalonMM(BusDriver):
-    """Avalon-MM Driver
+    """
+    Avalon Memory Mapped Interface (Avalon-MM) Driver
```

Comment:
Sometimes you add a new line after the opening triple quotes, sometimes you don't.  I've generally seen that the first line does have more than just the triple quotes.  I think the google style sphinx docstrings all begin on the triple quote line, so probably best to do that too.